data/plugins/toolbarview.lua

```diff
@@ -84,11 +84,12 @@ end
 
 function ToolbarView:on_mouse_pressed(button, x, y, clicks)
   local caught = ToolbarView.super.on_mouse_pressed(self, button, x, y, clicks)
-  if caught then return end
+  if caught then return false end
```

Comment:
Shouldn't this return `caught`?

Reply:
Yeah right.

Reply:
Also, @Guldoman , Should I return false here?https://github.com/Nightwing13/lite-xl/blob/60322a93a877d54bd451edfe61381ea8f44a9993/data/core/view.lua#L79
 (I am assuming it(the original function) is for scrolling). So either scroll or no scroll

Reply:
I don't think it's necessary as it returns `nil`, which is falsy.